src/compiler/objective_c_generator.cc

```diff
@@ -176,65 +176,71 @@ void PrintMethodImplementations(Printer *printer,
 
 string GetHeader(const ServiceDescriptor *service, const string prefix) {
   string output;
-  grpc::protobuf::io::StringOutputStream output_stream(&output);
-  Printer printer(&output_stream, '$');
+  {
+    // Scope the output stream so it closes and finalizes output to the string.
```

Comment:
Also, I'd be surprised if that isn't affecting more protoc code. I am fairly certain they have the same code pattern in there.